packages/cli/src/WebhookHelpers.ts

```diff
@@ -30,13 +30,14 @@ import {
 	IWebhookData,
 	IWebhookResponseData,
 	IWorkflowExecuteAdditionalData,
+	LoggerProxy,
```

Comment:
Maybe we do not need as much detail in the name when the logger is being used.

```suggestion
	LoggerProxy as Logger,
```